### PR TITLE
LPD-34483 - Change where clause on upgrade to filter data on DB side

### PR DESCRIPTION
**Pull request by G-Sa**
Related to: [LPD-34483](https://liferay.atlassian.net/browse/LPD-34483)

[LPD-34483]: https://liferay.atlassian.net/browse/LPD-34483?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-kaleo-service/src/main/java/com/liferay/portal/workflow/kaleo/internal/upgrade/v1_3_0/BaseUpgradeClassNames.java [hunk 1]
```diff
@@ -66,6 +66,8 @@ protected void updateWorkflowContext(
 		}
 	}
 
+	protected abstract String workflowContextWhereClause();
```
Comment (linolaoj) [inline modules/apps/portal-workflow/portal-workflow-kaleo-service/src/main/java/com/liferay/portal/workflow/kaleo/internal/upgrade/v1_3_0/BaseUpgradeClassNames.java:69]: Hi @G-Sa , can we rename it to `getWhereClause()`?

**Review by linolaoj — CHANGES_REQUESTED** [review]

**Review by linolaoj — APPROVED** [review]

**Comment by linolaoj** [comment]
ci:forward

**Comment by linolaoj** [comment]
ci:stop

**Comment by linolaoj** [comment]
@G-Sa, as we talked I'm closing this PR so you can forward it to the upgrade team.

**Comment by G-Sa** [comment]
Sent to the Database-Infraestructure team [here](https://github.com/liferay-database-infra/liferay-portal/pull/2258)

**Pull request closed without merging**